draft/2020-06-09-this-week-in-rust.md

```diff
@@ -20,6 +20,26 @@ Check out [this week's *This Week in Rust Podcast*]()
 
 ## News & Blog Posts
 
+* [Announcing Rust 1.44.0](https://blog.rust-lang.org/2020/06/04/Rust-1.44.0.html)
+* [From Rust to Svelte, what tech stack will I use](https://hugodaniel.pt/posts/browser-tech-stack-2020/)
+* [Graph & Tree Traversals in Rust](https://sachanganesh.com/programming/graph-tree-traversals-in-rust/)
+* [Programming languages: Rust enters top 20 popularity rankings for the first time](https://www.zdnet.com/article/programming-languages-rust-enters-top-20-popularity-rankings-for-the-first-time/)
+* [Programming Servo: Integrating Streams](https://medium.com/@polyglot_factotum/programming-servo-integrating-readablestream-1a7faebeeed7)
+* [A Rust SentencePiece implementation](https://guillaume-be.github.io/2020-05-30/sentence_piece)
+* [Rust Things I Miss in C](https://people.gnome.org/~federico/blog/rust-things-i-miss-in-c.html)
+* [RiB Newsletter #12 - ZK-Rustups](https://rustinblockchain.org/newsletters/2020-06-03-zk-rustups/)
+* [So What's Up with Microsoft's (and Everyone Else's) Love of Rust?](https://visualstudiomagazine.com/articles/2020/06/02/rust-love.aspx?m=1)
+* [The Story of Tail Call Optimizations in Rust](https://dev.to/seanchen1991/the-story-of-tail-call-optimizations-in-rust-35hf)
+* [Taking the Unhappy Path with Result, Option, unwrap and ? operator in Rust](https://dev.to/codeprototype/taking-the-unhappy-path-with-result-option-unwrap-and-operator-in-rust-482b)
+* [This Month in Rust OSDev (May 2020)](https://rust-osdev.com/this-month/2020-05/)
+* [This month in rustsim #11 (April - May 2020)](https://www.rustsim.org/blog/2020/06/01/this-month-in-rustsim/)
+* [Why the developers who use Rust love it so much](https://stackoverflow.blog/2020/06/05/why-the-developers-who-use-rust-love-it-so-much/?cb=1)
+* [Zero To Production #1: Setup - Toolchain, IDEs, CI](https://www.lpalmieri.com/posts/2020-06-06-zero-to-production-1-setup-toolchain-ides-ci/)
+* [Indonesian] [Berbagai alasan melakukan Programming dalam Rust](https://dev.to/rizki96/berbagai-alasan-melakukan-programming-dalam-rust-p67)
+* [paper] [Memory-Safety Challenge Considered Solved? An Empirical Study with All Rust CVEs](https://arxiv.org/abs/2003.03296)
```

Comment:
This is cool! I don't think it's necessary to label it as "paper" - it fits in quite well with this section!